scripts/make_exe.py

```diff
@@ -67,6 +67,13 @@ def build_cli():
     Generates an `.exe` file (with a related binaries directory) using `PyInstaller`.
     """
     _ra2ce_dir = Path(cli_module.__file__).parent
+    _logo = _root_dir.joinpath("docs", "_resources", "ra2ce_logo.ico")
+
+    if not _logo.exists():
+        from PIL import Image
+
+        img = Image.open(_logo.with_suffix(".png"))
+        img.save(_logo)
 
```

Comment:
In what case would you possibly not have a logo?

Reply:
We don't have an original `ico` file, we have a `.svg` and a `.png`, however for an exe icon you require the one we don't have. This is actually my fault because it should generate an `*ico` file.